src/python_pachyderm/mixin/pfs.py

```diff
@@ -287,13 +287,14 @@ def inspect_commit(
                     )
                 ]
             )
-        else:
+        elif uuid_re.match(commit):
             return self._req(
                 Service.PFS,
                 "InspectCommitSet",
                 commit_set=pfs_proto.CommitSet(id=commit),
                 wait=commit_state == pfs_proto.CommitState.FINISHED,
             )
+        return None
```

Comment:
You don't really need this, since Python automatically returns None

Reply:
We might actually want to raise an exception in this case

Reply:
added